### PR TITLE
Add occupancy entities to cu_inventario_legislatura

### PR DESCRIPTION
**Pull request by jbothma**

### CODE REVIEW

File: datasets/cu/inventario_legislatura/crawler.py [hunk 1]
```diff
@@ -61,5 +67,15 @@ def crawl(context: Context):
         entity.add("nationality", "cu")
         entity.add("topics", "role.pep")
 
+        occupancy = h.make_occupancy(
+            context,
+            entity,
+            position,
+            start_date=inception_date,
+            end_date=dissolution_date,
```
Comment (jbothma) [inline datasets/cu/inventario_legislatura/crawler.py:75]: make_occupancy will change these from current to ended in 2028

but @pudo can we assume this dataset is kept up to date and people who are no longer members are removed from the list? At least if they add an "ended" column we should get a warning from the audit.

Comment (pudo) [reply datasets/cu/inventario_legislatura/crawler.py:75]: No I think the page we're crawling is super specific to this congress. The reason for crawling it is also a bit diplomatic (they're friends). Maybe after that we should have a script for the parliament's web site?

**Comment by jbothma** [comment]
One option then is to explicitly say their status is "unknown", or do so
say 1 year after initial publication?

**Comment by jbothma** [comment]
@pudo how nasty would this be for this crawler?

```python

if settings.START_DATE > "2024-08-17":
    occupancy_status = OccupancyStatus.UNKNOWN
    context.log.warning("It's time to check if this page is getting updated")
else:
    occupancy_status = OccupancyStatus.CURRENT
```

**Comment by pudo** [comment]
Oh, I actually really like this. I kind of wonder if this could be done via reading `context.dataset.coverage.end` and checking if the current date is much newer - that would make it generalisable?

**Comment by jbothma** [comment]
I like that. Will do.

**Pull request merged**